client/js/admin.js

```diff
@@ -27,11 +27,11 @@ const submit = function() {
 
 	$("#status").text("Uploading...")
 
-	const XHR = new XMLHttpRequest()
-	XHR.open("POST", "")
-	XHR.setRequestHeader("Content-Type", "application/json")
+	const xhr = new XMLHttpRequest()
```

Comment:
I don't actually know or have an opinion on which it's supposed to be, but was following @bluepichu's comments on #123